cmsplugin_carousel_ai/models.py

```diff
@@ -1,9 +1,21 @@
 # -*- coding: utf-8 -*-
 from cms.models import CMSPlugin, force_text, Page
 from django.db import models
+from django.conf import settings
 from django.utils.translation import ugettext_lazy as _
 from filer.fields.image import FilerImageField
 
+def get_templates():
+    choices = [
+        ('carousel.html', _('Default')),
+    ]
+    choices += getattr(
+        settings,
+        'CAROUSEL_LINK_TEMPLATES',
```

Comment:
This should be probably called something like `CAROUSEL_PLUGIN_TEMPLATES` since these templates are options for the plugin.